src/components/types/GameType.tsx

```diff
@@ -0,0 +1,10 @@
+export type CellType = number | null;
+export type CellsType = CellType[][];
+export type HistoryType = CellsType[];
+export type DirectionType = 'up' | 'left' | 'right' | 'down';
+export type RotateDegreeType = 0 | 90 | 180 | 270;
```

Comment:
이것도 사소한건데 특별한 이유가 없다면 보통 타입 이름에 타입을 붙이진 않습니다!

```suggestion
export type Cell = number | null;
export type Cells = Cell[][];
export type History = Cells[];
export type Direction = 'up' | 'left' | 'right' | 'down';
export type RotateDegree = 0 | 90 | 180 | 270;
```